### PR TITLE
OP-353 - Update builds

### PR DESCRIPTION
**Pull request by SzymonKostrubiec**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -22,10 +22,10 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.1"]
-                symfony: ["^5.4", "^6.0"]
-                sylius: ["~1.12.0", "~1.13.0"]
-                node: ["^14.17.x"]
+                php: [ "8.1", "8.2", "8.3" ]
```
Comment (senghe) [inline .github/workflows/build.yml:25]: Sylius 1.12 has PHP ^8.0 rule.

File: .github/workflows/coding_standard.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [ "8.1", "8.2", "8.3" ]
```
Comment (senghe) [inline .github/workflows/coding_standard.yml:21]: Sylius 1.12 has PHP ^8.0 rule.

**Review by senghe — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (SzymonKostrubiec) [reply .github/workflows/build.yml:25]: We can't use PHP 8.0 because the code has constructors for PHP 8.1.

Comment (senghe) [reply .github/workflows/build.yml:25]: Good. I think we won't rewrite the constructors.

**Review by senghe — APPROVED** [review]

**Pull request merged**